src/pages/en/getting-started-new.md

```diff
@@ -0,0 +1,107 @@
+---
+setup: |
+    import Button from '../../components/Button.astro'
+layout: ~/layouts/MainLayout.astro
+title: Getting Started
+description: A basic intro to Astro.
+---
+Static Site Generator  🚀  Bring your own Framework  🚀  Ship Less JavaScript
+
+## Try Astro
+
+We've made it as easy as possible to get started with Astro either in your browser, or on your machine!
+
+### Online Playgrounds
+
+Visit [astro.new](https://astro.new) for the easiest way to "try before you buy." Choose from a variety of starter templates and start building a full, working version of Astro right in your browser!
+
+Or, **instantly launch our basic starter project** with just one click of a button:
+
+
+
+<div style="display: flex; flex-wrap: wrap; gap: 0.5rem;">
+    <Button href="https://astro.new/starter?on=codesandbox">Open in CodeSandbox</Button>
+    <Button href="https://astro.new/starter?on=stackblitz">Open in StackBlitz</Button>
+</div>
+
+### Install Astro Locally
+
+Ready to install?
+
+Get a new project up and running locally in no time with our easy `create-astro` CLI wizard!
+
+```bash
+# run this command in a new directory to get started!
+npm init astro
+```
+
+⚙️ Our [Installation Guide](/en/installation) has full, step-by-step instructions for installing Astro with your favourite package manager.
+
+⚙️ See instructions for [manual setup](/en/guides/manual-setup) instead.
+
+
+## Start building with Astro
+
+Jump right in and add some content and features to your site!
+
+🏗️ Add new [Astro (.astro) pages](/en/core-concepts/astro-pages) and/or [Markdown (.md) pages](/en/guides/markdown-content) to your site.
+
+🏗️ Create your first [Layout](/en/core-concepts/layouts).
+
+🏗️ Add additional [CSS and styling](/en/guides/styling) to your site. 
+
+*... see more How-To Guides*
+
+
+
+## Learn Astro
+
+See examples of some of the key concepts and patterns of an Astro site!
+
+📚 Read more about Astro’s [project structure.](/en/core-concepts/project-structure)
+
+📚 Learn more about Astro’s [built-in components.](/en/reference/builtin-components)
+
+📚 Explore Astro’s [API.](/en/reference/api-reference)
+
+*... see more reference material*
+
+## Combine with Astro
+
+Explore different integrations that our users have combined with Astro!
+
+🧰 Use a CMS with your Astro project.
+
+🧰 Set up eCommerce.
+
+🧰 Connect a database to your site.
+
+*... see more [third-party integrations](/en/integrations/integrations)*
+
+
+
+## Discuss Astro
+
+Join us in the [Astro Discord](https://astro.build/chat) to share with and get help from an active, friendly community!
+
+Say hi in our `#introduce-yourself` channel and let us know who you are!
```

Comment:
Can these "different things to do" be bullet points, and maybe shorter (<15 words each)?